### ISSUE TITLE
Dashboard - tabs have Broken Aria Menu tags 

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#149]
URL: https://rsams.dev.cloud.openplus.ca/en/user/
User Agent:  Chrome v99 + WAVE plug-in v3.16
Language: EN and FR pages

Issue Description: 
This broken aria-menu tags are enable users to use the arrows to navigate the tabs at the top of dashboard. See the error the screenshot.
![Dashboard - Broken Aria Menu](https://user-images.githubusercontent.com/88147013/161546607-46fe1fda-a67c-43c0-91fb-ee88108cb21b.jpg)
 

Steps to reproduce:
1. Login into RSAMs
2. While on the dashboard, turn on WAVE plugin 

Expected results:
Users should be able to use the arrow keys to navigate the tabs

**Comment by echouin** [comment]
Those are from Drupal. Issue reported by OP.

Also, if you are doing another a11y review, I would suggest that you get your list together first and prioritize. I'm being conscious of your support time.
